deploy/Dockerfile

```diff
@@ -18,11 +18,19 @@ FROM golang:1.10
 WORKDIR /go/src/github.com/GoogleContainerTools/kaniko
 COPY . .
 RUN make
+WORKDIR /usr/local/bin
```

Comment:
instead of changing workdir repeatedly can we ADD these into the right place?

Reply:
done!